recipes/booster/build.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+CXXFLAGS="$CXXFLAGS";
+CFLAGS="$CFLAGS";
+LDFLAGS="$LDFLAGS -Wl,-rpath ${PREFIX}/lib";
+CXX=g++;
+CC=gcc;
```

Comment:
Please don't change CXX or CC.

Reply:
> If you can get `make` to respect the compilers specified by `CC` and `CXX` then the builds should work.

Could you elaborate more on that? I've no experience with C programs. Also, I added your suggestions to the `meta.yaml` file and rolled back the `build.sh` file to a simple `make` call.

Reply:
It's mostly a matter of looking into the Makefile and seeing how it's defining which compiler to use. In many cases, it's defining a variable like `${CC}`, which makes life easy. In other cases the compiler may be hard-coded (e.g., `gcc -o foo foo.c`), which means you'll have to heavily modify the Makefile so it uses the correct compilers. I'll try to take a look at the make file used by this project and see if I can provide any pointers.